app/src/main/res/xml-cs/remote_config_defaults.xml

```diff
@@ -228,4 +228,8 @@
         <key>v2_efgsTravellerDefault</key>
         <value>false</value>
     </entry>
+    <entry>
+        <key>v2_howItWorksUITitle</key>
```

Comment:
Tenhle klíč jsem nenašel v Remote Configu, tzn. vypadá to že jsi nepoužil proces na zavádění nových klíčů a dáváš to sem manuálně. Všechny změny v RC by se měly dělat takto:
1) Přidáš to do rc.json v RC repu
2) Přeložíš to v OneSky nebo počkáš na překlad
3) Pomocí skriptu v RC repu to deployneš do RC
4) Spustíš skript v RC repu, který ti vygeneruje RC defaults XML a to dáš do PR